### DISCUSSION TITLE
بررسی تخصصی برنامه‌های گوشی

### DISCUSSION THREAD

**Discussion by Ahmadtaha1122** [community/community#65064] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

با سلام /چگونگی آپدیت device info گوشی ردمی و بروزرسانی سیستم و برنامه‌های گوشی

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Ahmadtaha1122 , Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!y

**Reply by Alimaleki1388** [reply]
چرا من مخازن حتی ۱درصد بازده نداشته  و حتی  حسابهام خالی خستن

**Reply by Alimaleki1388** [reply]
ببخشید اینو میگم من پولدار نیستم میخواستم پیشرفت کنم نابو. شوم حالا جز غرامت حسانم وبسته شدن همیشگی آن چی ینمیخ ام  و پا پس نمیکشم شما اعتبار دارید من هم نمیخوام لطمه بزنم

**Reply by Alimaleki1388** [reply]
من دیگه حرفی ر ای گفتن ندا م دیشب ثولار اشتراک گذاری کردم نیست حتی مدار کش  هم نیست چرا آخه  من حرفی برای گفتن ندارم خداحافط

**Comment by Alimaleki1388** [comment]
اگه حرفی بود بعد ازا دعای حقوقی من  خواهم شنید

**Comment by queenofcorgis** [comment]
Hi @Ahmadtaha1122 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
